test/utils/int/delay.ts

```diff
@@ -2,10 +2,12 @@ export async function delay(duration: Duration) {
     return new Promise(res => setTimeout(res, duration));
 }
 
+// values of TOAST_DOM_LOADED and TOAST_CLOSE durations are increased due to integration test issues,
+// to be investigated within the scope of ticket SERVICE-730
```

Comment:
Would put more concisely and follow convention around this. Something like

'// TODO: Investigate TOAST_DOM_LOADED and TOAST_CLOSE needing to be increaded for tests to pass on CI [SERVICE-730]`

Reply:
sure